index.js

```diff
@@ -28,11 +29,21 @@
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 3", "color: red") 
 
-// console.log(drinkWater(12))
-// // => "Man I sure am thirsty"
-// // => "Ahh that hits the spot"
-// // => 11
-// console.log("%c----------", "color: red") 
+
+
+function drink_water(current_thirst_level) {
+  console.log("Man I sure am thirsty")
+  current_thirst_level -= 1
+  console.log("Ahh that hits the spot")
+  current_thirst_level
+}
+
+
+console.log(drinkWater(12))
```

Comment:
Almost right - you just need to make sure to use the `return` keyword at the end of the function since JS doesn't have an implicit return, like Ruby does. You'd also want to use camelCase to follow JS naming conventions. Since you called it `drink_water`, the code on line 42 will not work. Make sure you have working code and fix any error messages so you can test out your code as you go.

```js
function drinkWater(currentThirstLevel) {
  console.log("Man I sure am thirsty")
  currentThirstLevel -= 1
  console.log("Ahh that hits the spot")
  return currentThirstLevel
}
```